Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/tracing/TracingData.kt

```diff
@@ -59,7 +59,8 @@ object TracingData {
                     lastExposureDetectionTime = Instant.now(),
                     allowManualUpdate = false,
                     daysWithEncounters = 0,
-                    activeTracingDays = 0
+                    activeTracingDays = 0,
+                    lastEncounterAt = Instant.now()
```

Comment:
`LowRisk` now also needs `lastEncounterAt` as we want to show this information in the card.